src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,91 @@
  */
 
 public class Plane {
+    private String name;
+    private String color;
+    private int speed;
+    private int capacity;
+    private int maxHeight;
 
+    public Plane(PlaneBuilder planeBuilder) {
```

Comment:
```suggestion
    private Plane(PlaneBuilder planeBuilder) {
```
explain why in the comment below

Reply:
yeah, it must be private. In other case we could create PlaneBuilder and with it's help change the  fields of our object.

Reply:
cause we should have access for this constructor only within this class so we won't be able to create Plane in other ways except PlaneBuilder.build()